src/lib.rs

```diff
@@ -492,6 +493,61 @@ pub struct BlitTarget {
     pub height: i32,
 }
 
+/// Mask specifying, which kinds of buffers to copy when blitting between two frame buffers.
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq)]
+pub struct BlitMask {
+
+    /// If the color buffer should be copied.
+    pub color: bool,
+
+    /// If the depth buffer should be copied.
+    pub depth: bool,
+
+    /// If the stencil buffer should be copied.
+    pub stencil: bool,
+}
+
+impl BlitMask {
+
+    /// Constructs a bit mask, that will only copy the color buffer
+    pub fn color() -> Self { BlitMask { color: true, depth: false, stencil: false} }
```

Comment:
Can you put the `BlitMask` onto a separate line? Also, I'd like a space before the }. Same goes for the other lines.